libethereum/Client.cpp

```diff
@@ -693,6 +692,9 @@ void Client::doWork(bool _doWait)
 
 	tick();
 
+	if (!isSealed)
```

Comment:
We might have slept inside `tick`, so I think it is better to re-assign isSealed here (of course inside a read guard for x_working).